src/material/core/common-behaviors/common-module.ts

```diff
@@ -106,7 +106,7 @@ export class MatCommonModule {
 
   /** Whether any sanity checks are enabled. */
   private _checksAreEnabled(): boolean {
-    return isDevMode() && !this._isTestEnv();
+    return (typeof ngDevMode === 'undefined' || ngDevMode) && !this._isTestEnv();
```

Comment:
Preliminary presubmit found that this change is breaking because `ngDevMode` is typed `boolean | null` inside Google, which is not assignable to `boolean`

Reply:
I've updated the type and fixed the error.